lib/licensed/sources/swift.rb

```diff
@@ -48,7 +53,7 @@ def pins
       end
 
       def dependency_path_for_url(url)
-        last_path_component = URI(url).path.split("/").last.sub(/\.git$/, "")
+        last_path_component = URI(url).path.split("/").last
```

Comment:
When Swift Package Manager clones Git repositories, it follows the same default behavior as the `git` client, where the resulting directory is named with the basename of the last path component. Wouldn't this change produce the wrong path for the cloned dependency?